runtime/doc/eval.txt

```diff
@@ -4856,7 +4861,12 @@ jobstart({cmd}[, {opts}])				{Nvim} *jobstart()*
 		   :call jobstart(['ping', 'neovim.io'])
 <		If it is a path (not a name), it must include the extension: >
 		   :call jobstart(['System32\ping.exe', 'neovim.io'])
-<
+
+<		NOTE: On Windows the List {cmd} is converted according to the
+		standard command line argument format (used in CommandLineToArgvW).
+		Some programs such as cmd.exe do not, and may behave
```

Comment:
I think this sentence is a bit confusing - some programs such as cmd.exe do not *what*?